bug-bounty-list/bug-bounty-list.json

```diff
@@ -8242,17 +8152,17 @@
   {
     "program_name": "Zerocopter",
     "policy_url": "https://www.zerocopter.com/en/responsible-disclosure",
-    "submission_url": "",
+    "submission_url": "https://app.zerocopter.com/rd/c83a1fda-b73a-4556-962e-a2bcacb8eb8d",
     "launch_date": "",
     "cash_reward": false,
     "swag": false,
     "hall_of_fame": false,
     "safe_harbor": "partial"
   },
   {
-    "program_name": "Zoho",
+    "program_name": "zoho",
     "policy_url": "https://bugbounty.zoho.com/bb/info#guide",
-    "submission_url": "",
+    "submission_url": "https://accounts.zoho.com/signin?servicename=BugBounty&serviceurl=https%3A%2F%2Fbugbounty.zoho.com%2Fsubmit",
```

Comment:
I think it's just "https://bugbounty.zoho.com/submit" (but that redirects you while unauth).